src/resolvers/Query/organizations.ts

```
@@ -2,52 +1,29 @@
+import { QueryResolvers } from "../../types/generatedGraphQLTypes";
```

ESLint

[error] 'QueryResolvers' is defined but never used.

(@typescript-eslint/no-unused-vars)